### DISCUSSION TITLE
Installation Issue with pySCHED

### DISCUSSION THREAD

**Discussion by AndrewOldnall** [jive-vlbi/sched#21] [Installation]
Hi there,
I've been tasked to use pySCHED for some work i'm doing for University but i'm coming across some errors when trying;
sudo pip3 install pythonSCHED

I'm using a UBUNTU shell on VMWARE. I have pip3 installed as well as 1.22 Numpy but when I run the above script I get errors which I have attached as a Notepad.

Any help would be great.
[SCHED problem.txt](https://github.com/jive-vlbi/sched/files/7835904/SCHED.problem.txt)

Edit: Made more info available on the link

**Comment by bmarcote** [comment]
Hi, sorry for the delay.
It seems that you do not have any Fortram compiler available in the system, so it is unable to compile part of the SCHED code.

(answered before in Issues, this is just for legacy)
